module_utils/ansible_nailgun_cement.py

```diff
@@ -41,6 +45,13 @@ class Organization(EntityCompareMixin, Organization):
 class Location(EntityCompareMixin, Location):
     pass
 
+class OperatingSystem(EntityCompareMixin, OperatingSystem):
+    pass
+
+
+class Architecture(EntityCompareMixin, Architecture):
+    pass
+
```

Comment:
We actually intend to remove EntityCompareMixin from cement and replace it with a more flexible comparison functionality:
In most of the cases we are so far comparing not ids, but actual content of entity-fields.
Thus we should maybe try and find a solution now